pyqt5_ui.py

```diff
@@ -0,0 +1,102 @@
+import sys
+from PyQt5.QtWidgets import QApplication, QMainWindow, QFrame, QMessageBox
+from game import Game
```

Comment:
imports should be grouped into 3 parts separated by a newline.
group1 = standard libs
group2 = external libs
group3 = project files

each group should have the direct imports first, followed by from imports in alphabetical order

Reply:
implemented